internal/dom/dom_suite_test.go

```diff
@@ -36,7 +36,7 @@ import (
 
 func TestDom(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "disrupt-o-meter internal suite")
+	RunSpecs(t, "disrupt-o-meter dom suite")
```

Comment:
What is the deeper reason for this name change? Because, the package was for the internals of the tool. Now the suite reads like "disrupt-o-meter" followed by "dom", which is the abbreviation of the previous name. Sounds like an unnecessary duplication. I would prefer to keep `internal` here to leave open the possibility that we might have a "public" package in the future.

Reply:
Yeah, it was a leftover of the time I looked for another project structure. 
I'll revert it completely now